### ISSUE TITLE
zip32 js child key derivation issue

### ISSUE THREAD

**Issue by uwuforever** [zcash/zcash#4305]
As a continuation of https://forum.zcashcommunity.com/t/help-with-zip32-child-keys/35806/3

Please help me figure out why child key derivation in my zip32 implementation doesn't match that of other zcash clients. 

https://github.com/uwuforever/zcash-zip32-js/blob/master/sapling.js

**Comment by daira** [comment]
This independent Python implementation that we use to generate test vectors may be of help: https://github.com/zcash-hackworks/zcash-test-vectors/blob/master/sapling_zip32.py

**Comment by uwuforever** [comment]
Thank you! Is this able to generate the initial key from the seed value? I'd like to write a derivation out step by step to figure out where mine went wrong.

**Comment by daira** [comment]
Yes, that's `ExtendedSpendingKey.master(seed)`. (It doesn't implement BIP 39-style word lists.)
